src/main/java/com/hubspot/jinjava/tree/ExpressionNode.java

```diff
@@ -51,7 +51,10 @@ public OutputNode render(JinjavaInterpreter interpreter) {
     if (interpreter.getConfig().isNestedInterpretationEnabled()) {
       if (
         !StringUtils.equals(result, master.getImage()) &&
-        (StringUtils.contains(result, "{{") || StringUtils.contains(result, "{%"))
+        (
+          StringUtils.contains(result, "{{") ||
```

Comment:
Looks like this should use the scanner symbols as well.

Reply:
I've made the changes. Although I'm not quite sure to if the `{{` is to be replaced by 
`TOKEN_PREFIX_CHAR + TOKEN_EXPR_START_CHAR`
or is it just 
`TOKEN_EXPR_START_CHAR + TOKEN_EXPR_START_CHAR`.

Please review it once again.
Cheers!